source/reference.txt

```diff
@@ -1,9 +1,53 @@
-.. place holder for the reference section TOC
-
 =========
 Reference
 =========
 
+:ref:`c2c-mongosync`
+   The primary process used in {+c2c-product-name+}.
+
+:ref:`c2c-config`
+   Configure :program:`mongosync` instances at startup using a
+   configuration file.
+
+:ref:`c2c-api`
+   API endpoints to interact with ``mongosync``. 
+
+:ref:`c2c-states`
+   States that ``mongosync`` enters depending on requests received. 
+
+:ref:`c2c-filtered-sync`
+   Inclusion filters to specify which databases and collections to
+   include in sync. 
+
+:ref:`c2c-oplog-sizing`
+   Increase the size of the oplog on the source cluster for large data
+   sets. 
+
+:ref:`c2c-failure-recovery`
+   Prepare for disaster recovery using :ref:`connections
+   <c2c-connecting>` between clusters.
+   
+:ref:`c2c-limitations`
+   Limitations of ``mongosync``. 
+
+:ref:`c2c-logging`
+   A running log of events, including entries such as the copying
+   collections and indexes, change stream events, and API calls. 
+
+:ref:`c2c-permissions-and-roles`
+   Required permissions on the source and destination clusters for the
+   user specified in the ``mongosync`` connection string. 
+
+:ref:`c2c-telemetry`
+   Anonymous, aggregated usage data to improve MongoDB products. 
+
+:ref:`c2c-verification`
+   Verify the successful transfer of your data before you switch your
+   application to the destination cluster.
+
+:ref:`c2c-release-version-numbers`
+   Understand the semantic versioning of {+c2c-product-name+}.
```

Comment:
```suggestion
   Understand the semantic versioning of ``mongosync``.
```
The semantic versioning described in the linked page applies only to the `mongosync` process. C2C is a broader term that encompasses things like Atlas Live Migrate.

Reply:
Done!